coordinator/src/services/translation/common/getResourcesOfType.ts

```diff
@@ -72,7 +72,7 @@ export function getLocations(bundle: fhir.Bundle): Array<fhir.Location> {
   return getResourcesOfType<fhir.Location>(bundle, "Location")
 }
 
-function getContainedResource<P extends fhir.Resource, C extends fhir.Resource>(
+function followContainedReference<P extends fhir.Resource, C extends fhir.Resource>(
```

Comment:
minor but I think in other places we have used `resolve{something}Reference` rather than follow